cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -164,7 +164,7 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<functions-single, single()>>                  | Predicate         | Returns true if the predicate holds for exactly one of the elements in a list.
 |<<functions-size, size()>>                      | Scalar            | Returns the number of items in a list.
 |<<functions-size-of-pattern-expression, size() applied to pattern expression>>  | Scalar   | Returns the number of sub-graphs matching the pattern expression.
-|<<functions-size-of-string, size() applied to string>>  | Scalar          | Returns the size of a string.
+|<<functions-size-of-string, size() applied to string>>  | Scalar          | Returns the number of Unicode code points in a string.
```

Comment:
Perhaps it would be easier for users to understand the term 'Unicode characters' instead of 'Unicode code points'?

Reply:
👍